apps/ucp-ui/src/Institutions/Institutions.tsx

```diff
@@ -114,6 +125,14 @@ const Institutions = () => {
   );
   const search = searchParams.get("search") || "";
 
+  const [sortBy, setSortBy] = useState(
```

Comment:
It seems like we could just store one sortBy value in state, and derive these other sortBy variables from the thing stored in state